app/lib/events/import_entity_event.rb

```diff
@@ -0,0 +1,37 @@
+module Events
+  class ImportEntityEvent
```

Comment:
Shall we refactor the common parts out of `EntityEvents`? Then we can rely on its specs too.

Reply:
We've deliberately avoided creating `Event` classes that aren't `Event` — clients can stick together whatever events they want using the `Event` interface

Reply:
This is coming in the next iteration as we want to get this out quickly.